### ISSUE TITLE
Delete list forbidden

### ISSUE THREAD

**Issue by NelsonGaldeman** [twisted1919/mailwizz-php-sdk#25]
I'm getting a forbidden error when I try to delete a list.

I confirmed It exists and I also tried to delete one manually and worked. I've tried with 50 different lists and I always got the same response.

    MailWizzApi_Params Object ( [_data:MailWizzApi_Params:private] => Array ( [status] => error [error] => Forbidden ) [_readOnly:MailWizzApi_Params:private] => )

**Comment by twisted1919** [comment]
Deletes are made via the `DELETE` method/http verb. Your server seems to have it disabled. Allow it and you will be fine.

**Comment by NelsonGaldeman** [comment]
Thank you for the fast replay. I went more in-deep about this matter and I found you already stated that hosting providers usually don't block those kind of _unused_ HTTP methods but unlikely that's far from truth.

Although being on your side by strictly following the standards I adivce you to change your mind on this and use GET/POST with extra information over data parameters for each instruction type. Just because of compatibiliy.

In my own case, I would have to switch my e-mail marketing provider because I don't even own the server or the shared hosting account. Unfortunally I can't right now, so that mean I can't use the API to delete/add(PUT) things.

I was able to get this HTTP header from my own hosting provider

    Allow: POST,OPTIONS,HEAD,GET,TRACE
